sql-select/actor-forty.sql

```diff
@@ -0,0 +1,3 @@
+select "actorId", "firstName", "lastName"

```

Comment:
This isn't all columns

---

![image](https://user-images.githubusercontent.com/16143771/130723989-be971eda-a821-4b4e-b6de-563eaf537aee.png)

Reply:
Fixed to pull all the column.

![image](https://user-images.githubusercontent.com/20628519/130725577-f8569097-73a8-4292-bbf0-8f958238bc5a.png)